### PR TITLE
Move x86 and mac builds to GitHub

### PR DESCRIPTION
**Pull request by janaknat**
Move i686, x86_64 and mac builds to Github to preserve Travis credits.

### CODE REVIEW

**Comment by bashtage** [comment]
How can we tell that it worked?

**Comment by janaknat** [comment]
@bashtage https://github.com/janaknat/statsmodels-wheels/actions/runs/411315059 . I ran it for my tree. For some reason, the i686 for all 3 python versions has 1 failure. Not sure if it is because I used the latest numpy, scipy and pandas.

**Comment by bashtage** [comment]
How do we pass the secrets needed to upload the wheels?

File: .github/workflows/build-wheels.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      UNICODE_WIDTH: 32
+      MB_PYTHON_VERSION: ${{ matrix.python-version }}
+      TRAVIS_PYTHON_VERSION: ${{ matrix.python-version }}
+      MB_ML_VER: ${{ matrix.MB_ML_VER }}
+      TRAVIS_REPO_SLUG: ${{ github.repository }}
+      TRAVIS_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST: ${{ github.event.number }}
+      TRAVIS_BUILD_DIR: ${{ github.workspace }}
+      SCIPY_WHEELS_NIGHTLY_ACCESS: ${{ secrets.SCIPY_WHEELS_NIGHTLY_ACCESS }}
+      MULTIBUILD_WHEELS_STAGING_ACCESS: ${{ secrets.MULTIBUILD_WHEELS_STAGING_BUILD }}
```
Comment (mattip) [inline .github/workflows/build-wheels.yml:40]: These are organization-wide secrets that allow uploading packages to anaconda.org

Comment (bashtage) [reply .github/workflows/build-wheels.yml:40]: Thanks.

**Pull request merged**